api/src/migrations/api/1611329857558-initialDataMetadataEntities.ts

```diff
@@ -0,0 +1,89 @@
+import {MigrationInterface, QueryRunner} from "typeorm";
+
+export class initialDataMetadataEntities1611329857558 implements MigrationInterface {
+
+    public async up(queryRunner: QueryRunner): Promise<void> {
+      await queryRunner.query(`
+      CREATE TYPE "status" AS ENUM (
```

Comment:
can we make the type name more descriptive, to make it clear whose kind of process/entity this is a status of.

also, is it this really an attribute of scenarios? in any case, we can refine things later, no need to be super exact here now.

Reply:
work_status